lib/resty/jwt.lua

```diff
@@ -806,7 +848,7 @@ function _M.verify_jwt_obj(self, secret, jwt_obj, ...)
     end
 
     local verified = false
-    local err = "verify error: reason unknown"
+    err = "verify error: reason unknown"
```

Comment:
why drop `local` here?

Reply:
It is already defined on `line 800` so `luacheck` was complaining about it.